source/documentation/deploying_services/s3.md

```diff
@@ -239,3 +239,96 @@ cf delete-service SERVICE_NAME
+
+### Create an IAM bucket policy that allows S3 bucket migration
+
+To start this migration, you need to apply an IAM bucket policy to your AWS account. This IAM policy will let the S3 broker copy objects from the PaaS S3 bucket to your S3 bucket. 
+
+Before creating the bucket policy, you’ll need the Amazon Resource Name (ARN) of the PaaS bucket’s owner. To get this ARN, [create a service key and get the service key credentials](#connect-to-an-s3-bucket-from-outside-of-the-govuk-paas).
+
+Export the credentials as AWS environment variables or create a profile in `~/.aws/credentials` to make the PaaS user’s credentials available to the AWS CLI.
+
+You now need the ARN of the AWS IAM user that created the PaaS S3 bucket. Run the following AWS CLI command:
+
+```
+% aws sts get-caller-identity --profile <NAME_OF_PROFILE_USED_IN_CREDENTIAL_FILE>
+```
+
+If you’ve already set the AWS credentials in the appropriate AWS environment variables, do not use the `--profile` option.
+
+If the command is successful, you’ll see an output that looks like this:
+
+```
+{
+   "UserId": "USER_ID",
+   "Account": "ACCOUNT_NAME",
+   "Arn": "arn:aws:iam::YYYYYYYYYY:user/paas-s3-broker/paas-s3-broker-ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ"
+}
+```
+
+In the IAM bucket policy, replace:
+
+* the `AWS` value with the `Arn` value from the output of your `aws sts get-caller-identity` command
+* the `arn:aws:s3:::` values with the `Arn` value of the bucket you created in your AWS account. You must not paste over the `/*` characters of the first item in the `Resource` list
+
+Your IAM bucket policy should look like this:
+
+```
+{
+   "Version": "2012-10-17",
+   "Statement": [
+       {
+           "Sid": "PaaS permissions",
+           "Effect": "Allow",
+           "Principal": {
+               "AWS": "<USER_ARN>"
+           },
+           "Action": [
+               "s3:ListBucket",
+               "s3:GetObject",
+               "s3:PutObject"
+           ],
+           "Resource": [
+               "arn:aws:s3:::<NAME_OF_TENANT_BUCKET>/*",
+               "arn:aws:s3:::<NAME_OF_TENANT_BUCKET>"
+           ]
+       }
+   ]
+}
+```
+
+You can [read more about attaching the IAM bucket policy to the appropriate bucket in the tenant's account](https://docs.aws.amazon.com/AmazonS3/latest/userguide/add-bucket-policy.html).
+
+### Synchronise the S3 buckets
+
+Run this command to synchronise the PaaS S3 bucket and the tenant’s S3 bucket:
+
+```
+aws s3 sync s3://<PAAS_BUCKET_NAME> s3://<TENANT_BUCKET_NAME> --source-region eu-west-1 --region <TENANT-BUCKET-REGION> --profile <NAME_OF_PROFILE_USED_IN_CREDENTIAL_FILE>
+```
+
+Replace:
+
+* `<PAAS_BUCKET_NAME>` with the `bucket_name` value provided in the output of the `cf service-key` command
+* `<TENANT_BUCKET_NAME>` with the `bucket_name` of the bucket that you created in your AWS account
+* `<TENANT_BUCKET_REGION>` with the `aws region` in which the bucket that you created in your AWS account resides
```

Comment:
```suggestion
* `<TENANT_BUCKET_REGION>` with the [AWS region](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html#Concepts.RegionsAndAvailabilityZones.Regions) of the bucket that you created in your AWS account
```

Does this work? I've linked out to the AWS docs but I'm not sure if it's the right place to point users. Is there somewhere in the PaaS docs we could link to that explains how to find out the AWS region?